docs/software/home_automation.md

```diff
@@ -167,4 +169,48 @@ The following hardware is required:
     Then click `Save`.  
     Press `F5` to refresh the page and view your energy consumption.
 
+## Domoticz - Multi platform Home Automation System
```

Comment:
Remember that we wanted to use the name as header only. But it's maybe good to stay consistent for now and remove the short descriptions from the headers all together in a single PR.

Reply:
I propose to keep it consistent for now and do a "global" change.